data/deep missions.txt

```diff
@@ -351,83 +360,86 @@ mission "Deep: Remnant 0"
 		has "Terminus exploration: done"
 	on offer
 		conversation
-			`While wandering through the spaceport, you are approached by a man who you instantly recognize as the scientist whos drone you recovered data from in Terminus thanks to his comically thick glasses. He introduces himself to you as Ivan and mentions how he heard you have been assisting others within the Deep.`
-			`	"I was hoping you would be able to continue assisting us in our research on the spatial anomaly in Terminus. Thanks to the data that you retrieved, we have been able to deduce that the anomaly is in fact some sort of wormhole, but it is far too unstable for anything to travel through it."`
-			`	"I must ask you, do you know of the Hai?" You nod. "Splendid! Then you will be able to help. One of the scientists on my team grew up in Hai space, and suggested that the so called Quantum Keystones which the Hai sell might have something to do with the wormhole, for there are some stories told in Hai space of these stones being able to unlock previously impassible pathways. May you retrieve ten of them for us so that we may be able to discover if these stones do allow access into the wormhole?"`
+			`While wandering through the spaceport, you are approached by a man you instantly recognize as the scientist whose drone you recovered data from in Terminus, thanks to his comically thick glasses. He introduces himself as Ivan and mentions he heard how helpful you have been to others within the Deep.`
+			`	"I was hoping you would continue assisting us in our research on the spatial anomaly in Terminus. Thanks to the data you retrieved, we have deduced that the anomaly is in fact some sort of wormhole, but it is far too unstable to support sending anything through it."`
+			`	"I must ask you, do you know of the Hai?" You nod. "Splendid! Then you will be able to help. One of the scientists on my team grew up in their territory, and suggested that the so called "Quantum Keystones" which they sell might have something to do with the wormhole. He recalls hearing some Hai folktales of these stones unlocking previously impassible pathways, soothing the minds and bodies of those who possessed them. Would you kindly retrieve ten of them for us, so that we may be able to discover if these legends are true, and they allow access into the wormhole?"`
 			branch remnant
 				has "First Contact: Remnant: offered"
 			choice
 				`	"I would be glad to help."`
-				`	"Sorry, but Hai space is way too far away for me to travel to."`
+				`	"Sorry, but Hai space is too far for me right now."`
```

Comment:
"right now" kind of assumes defer and not decline. I actually originally had "right now" in this choice but removed it so that it would be more clear that this is a declining option and not a defering one.